sdk/communication/Azure.Communication.AlphaIds/tests/Azure.Communication.AlphaIds.Tests.csproj

```diff
@@ -10,8 +10,10 @@
   <ItemGroup>
     <ProjectReference Include="$(AzureCoreTestFramework)" />
     <ProjectReference Include="..\..\Azure.Communication.Common\src\Azure.Communication.Common.csproj" />
-    <ProjectReference Include="..\..\Azure.Communication.Common\tests\Azure.Communication.Common.Tests.csproj" />
     <ProjectReference Include="..\src\Azure.Communication.AlphaIds.csproj" />
+    <Compile Include="$(AzureCoreSharedSources)ConnectionString.cs" LinkBase="Shared" />
+    <Compile Include="..\..\Shared\tests\AuthMethod.cs" LinkBase="Shared\Communication.Tests" />
+    <Compile Include="..\..\Shared\tests\CommunicationTestEnvironment.cs" LinkBase="Shared\Communication.Tests" />
```

Comment:
I would suggest adding <Compile> tag in the seperate <ItemGroup> as it's done for Chat & Identity libraries

Reply:
you mean a separate ItemGroup? yeah we can do it, but doesn't make much of a difference.

Reply:
you mean a separate ItemGroup? yeah we can do it, but doesn't make much of a difference.

Reply:
Yeah separate <ItemGroup>. (Somehow it got deleted in my comment). Yes, won't make any difference as such, but just to keep it clear that these are shared test classes and to be consistent across all other libraries.

Reply:
I agree that it doesn't influence the functionality but it's easy to make this change and the code will look more readable. I'll update